fiftyone/operators/executor.py

```diff
@@ -471,16 +472,20 @@ def view(self):
         if self._view is not None:
             return self._view
 
-        # Forces a dataset reload if necessary
```

Comment:
note here how we were relying on the fact that a `dataset.reload()` is triggered